src/main/scala/io/iohk/praos/domain/Environment.scala

```diff
@@ -1,7 +1,11 @@
 package io.iohk.praos.domain
 
+import io.iohk.praos.crypto.Seed
+
 case class Environment(initialStakeDistribution: StakeDistribution,
                        epochLength: Int,
+                       k: Int,
```

Comment:
What's this `k`? Maybe we could rename this variable or add a comment on it's purpose?

Reply:
I changed in the next PR! =)